darts-api-stub/transcriptions/transcriptions.js

```diff
@@ -505,6 +505,10 @@ router.get('/', (req, res) => {
 
 router.patch('/', (req, res) => {
   req.body.forEach((item) => {
+    // send 400 error for last requester transcription hide
+    if (item.transcription_id === 5) {
+      res.sendStatus(400);
```

Comment:
Few lines down below it will try send a status again which causes errors in express. You could update to something like below;

let status = 204;
  req.body.forEach((item) => {
    // send 400 error for last requester transcription hide
    if (item.transcription_id === 5) {
      status = 400;
    }
    const index = yourTranscriptionsStub.requester_transcriptions.findIndex(
      (x) => x.transcription_id == item.transcription_id
    );
    if (index >= 0) {
      yourTranscriptionsStub.requester_transcriptions.splice(index, 1);
    }
  });

  res.sendStatus(status);